paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetConfigurationKtx.kt

```diff
@@ -105,3 +113,42 @@ internal fun PaymentSheet.Appearance.parseAppearance() {
         )
     )
 }
+
+private fun PaymentSheet.Configuration.toVolatileConfiguration(): VolatilePaymentSheetConfiguration {
```

Comment:
Is it worth documenting what we're intentionally removing from these?

Reply:
Yeah, I can add a comment here about why.